gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql

```diff
@@ -0,0 +1,55 @@
+/* gpcontrib/arenadata_toolkit/arenadata_toolkit--1.2--1.3.sql */
+
+/*
+ * Returns column pair (table_schema, table_name) ordered by haven't been vacuumed
```

Comment:
It's very hard to read an understand this comment.
Can you try to split and rearrange it and the same one below?

Reply:
done